cmd/cloud/workspace.go

```diff
@@ -407,6 +407,7 @@ func newWorkspaceTeamAddCmd(out io.Writer) *cobra.Command {
 			return addWorkspaceTeam(cmd, args, out)
 		},
 	}
+	cmd.Flags().StringVarP(&workspaceID, "workspace-id", "w", "", "The Workspace's unique identifier")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&workspaceID, "workspace-id", "w", "", "workspace where you would like to add the team")
```
keeping it consistent with other workspace-id flag description in this file